docs/windows/latest-supported-vc-redist.md

```diff
@@ -0,0 +1,94 @@
+- **Windows XP Support**: Microsoft ended support for Windows XP on April 8, 2014. While the current version of the C++ Redistributable for Visual Studio 2015-2019 only supports Windows Vista, 7, 8.1, 10, and 11 older versions of the C++ Redistributable 2015-2019 can be installed on Windows XP. The last version of the C++ Redistributable for Visual Studio 2015-2019 that works on Windows XP is v14.24.28127.4. Download from [My.VisualStudio.com](https://my.visualstudio.com/) or directly using the following link corresponding to your architecture: [vc_redist.86.exe](https://download.visualstudio.microsoft.com/download/pr/bc2490f7-3f8e-4a5d-a61e-12b6d331713e/E59AE3E886BD4571A811FE31A47959AE5C40D87C583F786816C60440252CD7EC/VC_redist.x86.exe) or [vc_redist.x64.exe](https://download.visualstudio.microsoft.com/download/pr/3b070396-b7fb-4eee-aa8b-102a23c3e4f4/40EA2955391C9EAE3E35619C4C24B5AAF3D17AEAA6D09424EE9672AA9372AEED/VC_redist.x64.exe)
+
+## Visual Studio 2013 (VC++ 12.0)
+
+Below are the latest supported en-US Microsoft Visual C++ Redistributable packages for Visual Studio 2013.
+Downloaded additional versions and languages form [Update for Visual C++ 2013 Redistributable Package](https://support.microsoft.com/en-us/topic/update-for-visual-c-2013-redistributable-package-d8ccd6a5-4e26-c290-517b-8da6cfdf4f10) or from [My.VisualStudio.com](https://my.visualstudio.com/).
+
+| Architecture |   Version    |                                                    Link |
+| ------------ | :----------: | ------------------------------------------------------: |
+| X86          | 12.0.40664.0 | [vcredist_x86.exe](https://aka.ms/highdpimfc2013x86enu) |
+| X64          | 12.0.40664.0 | [vcredist_x64.exe](https://aka.ms/highdpimfc2013x64enu) |
+
+### Other versions
+
+- [Multibyte MFC Library for Visual Studio 2013](https://my.visualstudio.com/Downloads?pid=1430). This add-on for Visual Studio 2013 contains the multibyte character set (MBCS) version of the Microsoft Foundation Class (MFC) Library.
+- [Visual C++ 2013 Runtime for Sideloaded Windows 8.1 apps](http://download.microsoft.com/download/5/f/0/5f0f8404-9329-44a9-8176-ed6f7f746f25/vclibs_redist_packages.zip). For more information, see [C++ Runtime for Sideloaded Windows 8.1 apps] on the [VC++ Team Blog](https://devblogs.microsoft.com/cppblog/c-runtime-for-sideloaded-windows-8-1-apps/).
+
+## Visual Studio 2012 (VC++ 11.0) Update 4
+
+Below are the latest supported en-US Microsoft Visual C++ Redistributable packages for Visual Studio 2012 Update 4.
```

Comment:
You might want to phrase this as: "Below are the latest supported English (en-US)..."

Reply:
Fixed